### PR TITLE
Mobile version setup

### PR DESCRIPTION
**Pull request by Hirwacardinalloic**
## Description
 In this PR, we created a responsive Mobile version.  
### General requirements  
- [x] No linter errors. 
- [x] Follow **Github flow**. 
- [x] Professional Documentation.  
### Project requirements, We:  
- [x]  Added HTML, CSS 

 # Happy reviewing 😃

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,5 +48,256 @@ <h4 class="connection">LET'S CONNECT</h4>
...
+                  </p>
+                </div>
+                <div class="language-list-holder">
+                  <ul class="language-container">
+                    <li class="language-item"><span>JavaScript</span></li>
+                    <li class="language-item"><span>Html</span></li>
+                    <li class="language-item"><span>Css</span></li>
+                  </ul>
+                </div>
+                <button type="button" class="butto">See project</button>
```
Comment (wuletawwonte) [inline index.html:85]: - [ ] Please add button interaction styling for the `See project` button. You can follow the button styling: Active, Hover and Pressed
Buttons(1)

File: index.html [hunk 2]
```diff
@@ -48,5 +48,256 @@ <h4 class="connection">LET'S CONNECT</h4>
...
+              </section>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="about-me">
+        <div class="about-me-holder">
+          <section>
+            <h2 class="head">About Myself</h2>
+            <p class="helloo">
```
Comment (wuletawwonte) [inline index.html:211]: - [ ] Please fix the dark gray background color in the top right corner of the about me section. The background should be similar to the upper part of the website. 
imageedit_3_7553973452

**Review by wuletawwonte — CHANGES_REQUESTED** [review]
# Hi @Hirwacardinalloic  👋🏿👋🏿👋🏿  

Good job so far!
There are some issues that you still need to work on to go to the next project! You have only a few more steps and you are done! 💪 💪 💪 

giphy

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Hirwacardinalloic** [comment]
hello,
I guess we're not supposed to make hover on the button...
but if so which color do you want me to use because am not seeing it on the
Figma

**Review by DammyShittu — COMMENTED** [review]
Hi @Hirwacardinalloic,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

#### You have submitted a Pull Request with failing linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**